sandbox/package.json

```diff
@@ -7,7 +7,7 @@
     "check-licenses": "node_modules/.bin/license-checker --failOn GPL --failOn LGPL --failOn AGPL",
     "start": "node index.js",
     "start:hotreload": "nodemon index.js",
-    "test": "node ./node_modules/.bin/nyc node ./node_modules/.bin/mocha --reporter mocha-junit-reporter --reporter-options mochaFile=./sandbox-unit-report.xml"
```

Comment:
This may break the test suite on other peoples computers, its better to directly call the local binary than to use the one in $PATH - that could be the global one rather than the specific one we have installed